Source/Diagnostics/ComputeDiagFunctors/ComputeDiagFunctor.H

```diff
@@ -33,11 +33,11 @@ public:
     virtual void PrepareFunctorData ( int i_buffer, bool ZSliceInDomain,
```

Comment:
Can you please add Doxygen for the arguments of `PrepareFunctorData`?

Since this is a virtual API implemented by many compute functors and half of them became BTD specific, it's probably best if we describe them.

Reply:
Done :)